netcore/Makefile

```diff
@@ -76,12 +81,12 @@ corefx/.stamp-dl-corefx-tests-$(NETCORETESTS_VERSION):
 	touch $@
 
 run-sample: prepare
-	dotnet build sample/HelloWorld
+	$(DOTNET) build sample/HelloWorld
 	MONO_ENV_OPTIONS="--debug" COMPlus_DebugWriteToStdErr=1 ./dotnet --fx-version "$(NETCOREAPP_VERSION)" sample/HelloWorld/bin/Debug/netcoreapp3.0/HelloWorld.dll
```

Comment:
put $(DOTNET) here in replace of ./dotnet

Reply:
@steveisok `./dotnet` refers to a local runtime we download and where we replace clrjit and mscorlib. I guess we will get rid of it eventually but not now

Reply:
Ok... I was assuming we would need to refer to local dotnet across the board.

Reply:
$(DOTNET) is a complete SDK+Runtime with version specified in global.json.
./dotnet is a runtime with version = NETCOREAPP_VERSION